otelcolbuilder/.otelcol-builder.yaml

```diff
@@ -115,8 +115,6 @@ replaces:
   # TODO: remove this when:
   # - regexp log filtering is released upstream:
   #   PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/5680
-  # - record level log filtering is available upstream:
-  #   commit: https://github.com/pmalek-sumo/opentelemetry-collector-contrib/commit/11de2c03d3dbbde2e73b8a816318a2515c843985
```

Comment:
This is already merged https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/5418 and available upstream https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/cb2a35d282351a071707c76448632fd7ffebc963 since `0.37.0`